src/content/en/fundamentals/media/mobile-web-video-playback.md

```diff
@@ -0,0 +1,611 @@
+orientation change, let's unlock screen with `screen.orientation.unlock()` if
+user holds device in portrait mode and screen is locked in landscape mode.
+
+<pre class="prettyprint">
+function lockScreenInLandscape() {
+  if (!('orientation' in screen)) {
+    return;
+  }
+  // Let's force landscape mode only if device is in portrait mode and can be held in one hand.
+  if (matchMedia('(orientation: portrait) and (max-device-width: 768px)').matches) {
+    screen.orientation.lock('landscape')
+    <strong>.then(function() {
+      listenToDeviceOrientationChanges();
+    })</strong>;
+  }
+}
+</pre>
+
+    function listenToDeviceOrientationChanges() {
+      if (!('DeviceOrientationEvent' in window)) {
+        return;
+      }
+      var previousDeviceOrientation, currentDeviceOrientation;
+      window.addEventListener('deviceorientation', function onDeviceOrientationChange(event) {
+        // event.beta represents a front to back motion of the device and
+        // event.gamma a left to right motion.
+        if (Math.abs(event.gamma) > 10 || Math.abs(event.beta) < 10) {
+          previousDeviceOrientation = currentDeviceOrientation;
+          currentDeviceOrientation = 'landscape';
+          return;
+        }
+        if (Math.abs(event.gamma) < 10 || Math.abs(event.beta) > 10) {
+          previousDeviceOrientation = currentDeviceOrientation;
+          // When device is rotated back to portrait, let's unlock screen orientation.
+          if (previousDeviceOrientation == 'landscape') {
+            screen.orientation.unlock();
+            window.removeEventListener('deviceorientation', onDeviceOrientationChange);
+          }
+        }
+      });
+    }
+
+As you can see, this is the seamless fullscreen experience we were looking for.
+To see this in action, check out the [sample]{: .external }.
+
+<video controls controlsList="nodownload" muted playsinline>
+  <source src="/web/fundamentals/media/videos/perfect-fullscreen.webm"
+          type="video/webm; codecs=vp8">
+  <source src="/web/fundamentals/media/videos/perfect-fullscreen.mp4"
+          type="video/webm; codecs=mp4">
+</video>
+
+## Background playback
+
+When you detect a web page or a video in the web page is not visible anymore,
+you may want to update your analytics to reflect this. This could also affect
+the current playback as in picking a different track, pause it, or even show
+custom buttons to the user for instance.
+
+### Pause video on page visibility change
+
+With the [Page Visibility API], we can determine the current visibility of a
+page and be notified of visibility changes. Code below pauses video when page
+is hidden. This happens when screen lock is active or when you switch tabs for
+instance.
+
+    document.addEventListener('visibilitychange', function() {
+      // Pause video when page is hidden.
+      if (document.hidden) {
+        video.pause();
+      }
+    });
+
+Note: Chrome for Android already pauses videos when page is hidden.
```

Comment:
Maybe also mention that most mobile browsers offer controls outside of the browser that allow to resume a paused video so this technique should not be used if the user is not allowed to play in the background?